src/azure-cli/azure/cli/command_modules/vm/aaz/profile_2020_09_01_hybrid/sig/_delete.py

```diff
@@ -19,9 +19,9 @@ class Delete(AAZCommand):
     """
 
     _aaz_info = {
-        "version": "2021-10-01",
+        "version": "2019-12-01",
```

Comment:
Why do we downgrade the api-version here?

Reply:
It should be `2019-12-01` instead of `2021-10-01`, and this PR just fix it.